vllm/spec_decode/ngram_worker.py

```diff
@@ -37,7 +38,7 @@ def init_device(self):
 
         # Current only support Top1Proposer
         self._proposer = Top1Proposer(
-            self,
+            weakref.proxy(self),
```

Comment:
This is from https://github.com/vllm-project/vllm/pull/4737. it is needed to run spec decoding test locally for debugging